command/server.go

```diff
@@ -100,19 +100,25 @@ func (c *ServerCommand) Run(args []string) int {
 	// start logging too early.
 	c.logGate = &gatedwriter.Writer{Writer: colorable.NewColorable(os.Stderr)}
 	var level int
-	logLevel = strings.ToLower(strings.TrimSpace(logLevel))
+	var logLevel string
+	if os.Getenv("VAULT_LOG_LEVEL") != "" {
+		logLevel = os.Getenv("VAULT_LOG_LEVEL")
+	}
+	if logLevelFlag != "" {
```

Comment:
generally env vars take precedence over config values, think this should too?

Reply:
This isn't a config value, it's a flag -- which generally takes precedence over env vars :-)